src/core.c

```diff
@@ -25,6 +25,8 @@
 *   #define PLATFORM_RPI
 *       Windowing and input system configured for Raspberry Pi i native mode (no X.org required, tested on Raspbian),
 *       graphic device is managed by EGL and inputs are processed is raw mode, reading from /dev/input/
+*   #define USERAWKEYBOARD
```

Comment:
The other naming conventions use `PLATFORM_RPI`, with an underscore between the words. Should it be `USE_RAW_KEYBOARD`? Could even go one step further and adopt the `SUPPORT_` prefix.

Reply:
Yes, that's right, actually, still thinking what to do with that flag... considering no exposing it directly to the user and default to the new keyboard input system... actually old one had some bugs...